packages/components/scripts/wrapper-generator/NextJsReactWrapperGenerator.ts

```diff
@@ -3,9 +3,16 @@ import { ReactWrapperGenerator } from './ReactWrapperGenerator';
 import type { ExtendedProp } from './DataStructureBuilder';
 import { pascalCase, paramCase } from 'change-case';
 import { getComponentMeta } from '@porsche-design-system/component-meta';
+import { INTERNAL_TAG_NAMES } from '@porsche-design-system/shared';
 
 export class NextJsReactWrapperGenerator extends ReactWrapperGenerator {
   protected projectDir = 'react-ssr-wrapper';
+  protected ignoreComponents: TagName[] = INTERNAL_TAG_NAMES.filter(
+    (tagName) => tagName !== 'p-select-wrapper-dropdown'
+  );
+  protected unexposedComponentTagNames: TagName[] = INTERNAL_TAG_NAMES.filter(
```

Comment:
Is this correct?
If `unexposedComponentTagNames` equals `INTERNAL_TAG_NAMES` as a default I would expect that `p-select-wrapper-dropdown` is a part of it and therefore won't be exposed as part of the barrel file.

Reply:
Thats why im filtering it out of the array?